src/Umbraco.Infrastructure/Runtime/FileSystemMainDomLock.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Umbraco.Cms.Core.Hosting;
+using Umbraco.Cms.Core.Runtime;
+
+namespace Umbraco.Cms.Infrastructure.Runtime
+{
+    internal class FileSystemMainDomLock : IMainDomLock
+    {
+        private readonly ILogger<FileSystemMainDomLock> _log;
+
+        private readonly CancellationTokenSource _cancellationTokenSource = new();
+
+        private readonly string _lockFilePath;
+        private readonly string _releaseSignalFilePath;
+
+        private FileStream _lockFileStream;
+
+        public FileSystemMainDomLock(
+            ILogger<FileSystemMainDomLock> log,
```

Comment:
Everywhere else we call it logger and _logger

Reply:
resolved 4e38e299